src/main/java/redis/clients/jedis/BasicCommands.java

```diff
@@ -41,4 +41,6 @@
   String configResetStat();
 
   Long waitReplicas(int replicas, long timeout);
+
+  String readonly();
```

Comment:
@taer I'm not saying that `JedisCluster` should have the `readonly` command. I'm suggesting to move this `readonly` signature to `ClusterCommands` interface which is implemented by `Jedis` class not `JedisCluster`